### ISSUE TITLE
validation sets

### ISSUE THREAD

**Issue by topepo** [tidymodels/stacks#24]
With `rsample`, validation sets are structured a little bit differently. You never get hold-out predictions on the entire training set, just the assessment set. When we assemble the data stack, we can't just substitute that data into the previous resampling strategy since we only have assessment set predictions (and that isn't helpful for tuning the `glmnet` model). 

I suggest creating a brand new bootstrap object for the data when, originally, a validation set was used (like you originally had).

**Comment by simonpcouch** [comment]
Was this [fixed](https://github.com/tidymodels/stacks/blob/main/R/stack_blend.R#L120-L131) in #25?
